extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -96,7 +99,14 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	};
 
 	GoogleTag.prototype.push = function (callback) {
-		googleTag.cmd.push(callback);
+		if (this.storeCommands) {
+			cmd.push(callback);
+		}
+		window.googletag.cmd.push(callback);
```

Comment:
Do we want to add callback to `window.googletag.cmd` even if `this.storeCommands` is set and we add the same callback there?